code/modules/mob/new_player/new_player.dm

```diff
@@ -254,6 +259,11 @@
 
 /mob/new_player/proc/create_character(turf/spawn_turf)
 	spawning = TRUE
+	if(client.prefs.organ_data[BP_CHEST] == "cyborg")
+		if(!whitelist_lookup(SPECIES_IPC, client.ckey) && client.prefs.species != SPECIES_IPC)
```

Comment:
Replace SPECIES_IPC to SPECIES_FBP and define with

Reply:
ok, I should test it again